core/src/main/java/io/questdb/cairo/mv/MatViewDefinition.java

```diff
@@ -48,43 +48,64 @@
 import static io.questdb.std.datetime.microtime.Timestamps.MINUTE_MICROS;
 
 public class MatViewDefinition implements Mutable {
-    public static final int INCREMENTAL_REFRESH_TYPE = 0;
-    public static final int INCREMENTAL_TIMER_REFRESH_TYPE = 1;
+    public static final int IMMEDIATE_REFRESH_TYPE = 0;
```

Comment:
Can you please add comments with SQL examples of what is IMMEDIATE vs MANUAL refresh type?